.vscode/launch.json

```diff
@@ -24,20 +24,13 @@
             "preLaunchTask": "watch"
         },
         {
-            "name": "Attach to ASL Server",
-            "type": "node",
-            "request": "attach",
-            "port": 6009,
-            "restart": true,
-            "outFiles": ["${workspaceRoot}/dist/src/stepFunctions/asl/**.js"]
-        },
-        {
-            "name": "Attach to SSM Document Language Server",
-            "type": "node",
-            "request": "attach",
-            "port": 6010,
-            "restart": true,
-            "outFiles": ["${workspaceRoot}/dist/src/ssmDocument/ssm/ssmServer.js"]
+            "name": "Extension (web)",
```

Comment:
- names related things with common prefix to improve discoverability
- move ASL/SSM stuff near the bottom, it's rarely needed

<img width="348" alt="image" src="https://github.com/aws/aws-toolkit-vscode/assets/55561878/9b8ca7e1-b196-4775-8f11-ca900320c17d">